static/app/components/searchSyntax/parser.tsx

```diff
@@ -697,14 +697,15 @@ export class TokenConverter {
   });
 
   tokenValueText = (value: string, quoted: boolean) => {
-    const isContains = value.startsWith('*') && value.endsWith('*');
+    // We only want to consider a value to be `contains` if it is at least one character being wrapped in `*`
+    const isContains = value.length > 2 && value.startsWith('*') && value.endsWith('*');
```

Comment:
Had to make this change as it was parsing `*` as contains, which is not what we want to do. After chatting with @doralchan when users enter multiple `*`'s we want to concatenate into a single `*`.

Reply:
Created a ticket for this in linear